src/main/java/org/apache/commons/crypto/utils/Utils.java

```diff
@@ -57,17 +57,17 @@
   private static final int AES_BLOCK_SIZE = AES_CTR_NOPADDING.getAlgorithmBlockSize();
 
   static {
-    loadChimeraSystemProperties();
+    loadCommonsCryptoSystemProperties();
```

Comment:
Suggest to remove the string like "CommonsCrypto" from the method name for simplicity.